README.md

```diff
@@ -2,34 +2,42 @@
 
 - Bonding provides a method for aggregating multiple network interfaces into a single logical &quot;bonded&quot; interface.
 - According to the 802.3ad specification, Linux Bonding drivers provides various flavours of bonded interfaces depending on the mode (bonding policies), such as round robin, active aggregation
-- When Bonding CNI configured as a standalone plugin, physical interfaces are obtained from host network namespace. With these physical interfaces a bonded interface is created in container network namespace.
-- A major user case for bonding in containers is network redundancy of an application in the case of network device or path failure and unavailability. For more information - refer to [network redundancy using interface bonding](https://www.howtoforge.com/tutorial/how-to-configure-high-availability-and-network-bonding-on-linux/)
+- When Bonding CNI is configured as a standalone plugin, interfaces are obtained from the host network namespace. With these physical interfaces a bonded interface is created in the container network namespace.
+- When used with [Multus](https://github.com/intel/multus-cni) users can bond two interfaces that have previously been passed into the container.
+- A major use case for bonding in containers is network redundancy of an application in the case of network device or path failure and unavailability. For more information - refer to [network redundancy using interface bonding](https://www.howtoforge.com/tutorial/how-to-configure-high-availability-and-network-bonding-on-linux/)
 - And for more information on the bonding driver please refer to [kernel doc](https://www.kernel.org/doc/Documentation/networking/bonding.txt)
 
-## Build &amp; Clean
+## Build
 
-This plugin is recommended to be built with Go 1.7.5 which has been fully tested.
+It is recommended that Bond CNI be built with Go 1.7.5 which has been fully tested. Later versions of Go should also be compatible but have not been validated.
+
+- Build the source code to binary:
 
-- Build the source codes to binary:
 ```
 #./build
 ```
+
 - Copy the binary to the CNI folder for the testing:
+
 ```
 # cp ./bin/bond /opt/cni/bin/
 ```
+
+The binary should be placed at /opt/cni/bin on all nodes on which bonding will take place. That is all nodes to which a container with a bonded interface can be deployed.
+
 ### Network configuration reference
 
 - name (string, required): the name of the network
 - type (string, required): &quot;bond&quot;
 - ifname (string, optional): name of the bond interface
 - miimon (int, required): specifies the arp link monitoring frequency in milliseconds
+- linksInContainer(boolean, optional): specifies if links are in container to start. Default is false i.e. look for interfaces on host before bonding.
```

Comment:
maybe change `links` to `slave links` for better understanding.

Reply:
I feel like we may want to change the default value to `True`  because in SR-IOV case, we'd expect device IDs be passed down from Multus instead of specifying then directly in the NAD.

Reply:
This is a good change we can make in a future commit.

Reply:
Wording change made to slave link